sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,197 @@
+  FROM
+    mozdata.static.third_party_standardized_country_names
+),
+stripe_subscriptions_history AS (-- CTE for all_subscriptions
+  SELECT
+    *,
+    CONCAT(
+      subscription_id,
+      COALESCE(
+        CONCAT(
+          "-",
+          NULLIF(ROW_NUMBER() OVER (PARTITION BY subscription_id ORDER BY valid_from), 1)
+        ),
+        ""
+      )
+    ) AS subscription_sequence_id
+  FROM
+    `mozdata.subscription_platform.stripe_subscriptions_history`
+  WHERE
+    -- Only include the current history records and the last history records for previous plans.
+    (valid_to IS NULL OR plan_ended_at IS NOT NULL)
+    AND status NOT IN ("incomplete", "incomplete_expired")
+),
+stripe_subscriptions AS (-- CTE for all_subscriptions
+  SELECT
+    -- user_id,
+    customer_id,
+    subscription_sequence_id AS subscription_id,
+    IF(
+      subscription_sequence_id != subscription_id,
+      subscription_id,
+      NULL
+    ) AS original_subscription_id,
+    plan_id,
+    status,
+    synced_at AS event_timestamp,
+    IF(
+      (trial_end > TIMESTAMP(CURRENT_DATE) OR ended_at <= trial_end),
+      NULL,
+      COALESCE(plan_started_at, subscription_start_date)
+    ) AS subscription_start_date,
+    --first subscription start date associated with the subscription id
+    IF(
+      (trial_end > TIMESTAMP(CURRENT_DATE) OR ended_at <= trial_end),
+      NULL,
+      subscription_start_date
+    ) AS original_subscription_start_date,
+    IF(plan_started_at IS NOT NULL, "Plan Change", NULL) AS subscription_start_reason,
+    -- created,
+    trial_start,
+    trial_end,
+    canceled_at,
+    canceled_for_customer_at,
+    cancel_at,
+    cancel_at_period_end,
+    COALESCE(plan_ended_at, IF(ended_at < TIMESTAMP(CURRENT_DATE), ended_at, NULL)) AS ended_at,
+    IF(plan_ended_at IS NOT NULL, "Plan Change", NULL) AS ended_reason,
+    -- fxa_uid,
+    country,
+    country_name,
+    -- user_registration_date,
+    -- entrypoint_experiment,
+    -- entrypoint_variation,
+    -- utm_campaign,
+    -- utm_content,
+    -- utm_medium,
+    -- utm_source,
+    -- utm_term,
+    provider,
+    plan_amount,
+    -- billing_scheme,
+    plan_currency,
+    plan_interval,
+    plan_interval_count,
+    plan_interval_timezone,
+    product_id,
+    product_name,
+    CONCAT(
+      plan_interval_count,
+      "-",
+      plan_interval,
+      "-",
+      plan_currency,
+      "-",
+      (plan_amount / 100)
+    ) AS pricing_plan,
+    -- Stripe billing grace period is 7 day and Paypal is billed by Stripe
+    INTERVAL 7 DAY AS billing_grace_period,
+    promotion_codes,
+    promotion_discounts_amount,
+  FROM
+    stripe_subscriptions_history
+  LEFT JOIN
+    standardized_country
+  USING
+    (country)
+  -- LEFT JOIN
+  --   users
+  -- USING
+  --   (fxa_uid)
+  -- LEFT JOIN
+  --   attribution
+  -- USING
+  --   (fxa_uid)
+  WHERE
+    (product_name = "Relay Premium" OR product_name = "Mozilla VPN & Firefox Relay")
```

Comment:
Relying on the product names could be fragile as they could be changed arbitrarily.  I'd suggest checking the capabilities metadata like the VPN ETL does:
```suggestion
    "premium-relay" IN UNNEST(stripe_subscriptions_history.product_capabilities)
    OR "premium-relay" IN UNNEST(stripe_subscriptions_history.plan_capabilities)
```

Reply:
Done. I tried to use the tables that I have access to and `stripe_subscriptions_history` is one of those tables that I can't access.

Reply:
I'm pretty sure you should have access to query `subscription_platform.stripe_subscriptions_history`.  If you can successfully test run this query as a whole then you have access.

Reply:
You are right. I just confirmed that I do have the access. So just to clarify for  future reference, In `subscription_platform.stripe_subscriptions_history`, I see `relay-phones` and `premium-relay` values in both `plan_capabilities` and `product_capabilities`. Shouldn't `relay-phones` be included in here?

Reply:
Sure, you could also include `relay-phones`, but as far as I'm aware the phone-masking feature is always in addition to the premium email-masking feature, so there (currently) would never be a product/plan that had `relay-phones` without `premium-relay`.